src/Pim/Bundle/CatalogBundle/Elasticsearch/Sorter/InGroupSorter.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Pim\Bundle\CatalogBundle\Elasticsearch\Sorter;
+
+use Pim\Component\Catalog\Query\Sorter\FieldSorterInterface;
+
+/**
+ * @author    Philippe Mossière <philippe.mossiere@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+class InGroupSorter extends BaseFieldSorter implements FieldSorterInterface
+{
+    const GROUPS_KEY = 'groups';
```

Comment:
INLINE PLEASE, ARE YOU SURE YOU WANT TO SUPPORT ANY BC ON THIS CLASS ? IT IS PUBLIC GOD DAMMIT !! Can't stand this damn. :trollface: